functions/utils/getColor.ts

```diff
@@ -0,0 +1,13 @@
+import ColorThief from 'colorthief/src/color-thief-node'
```

Comment:
🤔 may steal this for personal projects

Reply:
its solid! i would recommend this [library](https://github.com/Vibrant-Colors/node-vibrant) over color-thief, but i had some issues with it and cloudflare so i was forced to use color-thief